api.yaml

```diff
@@ -184,6 +184,10 @@ components:
             independent results.
           items:
             $ref: '#/components/schemas/AssessmentResult'
+        status:
+          type: string
+          example: vulnerable
+          description: status of a vulnerability, can be used to filter on
```

Comment:
nit: ending a sentence with a preposition. I think "can be used for filtering" reads a bit more naturally. NBD either way!!